### PR TITLE
Improvements related to repeated fields packing

### PR DESCRIPTION
**Pull request by Clement-Jean**
Repeated fields on complex objects (user defined, string, bytes) produce encoding overhead (repeat field id and type as many times as there are values). Furthermore, repeated field keep the order of object. Thus we can make multiple repeated fields, the rssis and channels field will be packed (no overhead) and only the ssids will have overhead. In the end, we can reconstruct objects with a zip of all the lists.

### CODE REVIEW

File: compare.js [hunk 1]
```diff
@@ -12,34 +12,42 @@ const root = await protobuf.load(path.join(process.cwd(), "sitesurvey.proto"));
...
+      && chan != undefined
+      && ssid != undefined
+      && rssi != undefined
+})
+
 // Create the protobuf message, convert Mac addresses to long to further save data
 // We don't have to use short keys (v, chan) for the protobuf message (as in the JSON)
 // and can use full, descriptive names.
 const message = siteSurvey.create({
-  ...jsonData,
```
Comment (coderbyheart) [inline compare.js:19]: This removes the `timestamp`.

**Comment by coderbyheart** [comment]
Thanks, this is great, I've learned that when the same values appear multiple times we can safe more bytes using the `repeated` keyword!

File: compare.js (hunk 1 above)
Comment (Clement-Jean) [reply compare.js:19]: My bad, I'm going to add that later, but the result shouldnt change much.

**Comment by coderbyheart** [comment]
Thanks again, but it was my bad that I only later realized that some fields are optional. That's why this approach won't work.

**Pull request closed without merging**